src/screens/profile/CreateAmenities/CreateAmenities.tsx

```diff
@@ -0,0 +1,193 @@
+      setOtherAmenities(data.otherAmenities.split(' ,'));
+    } else {
+      throw new Error('Error while fetching amenities');
+    }
+  };
+
+  useEffect(() => {
+    if (!isNew) {
+      getSavedAmenities();
+    }
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, []);
+
+  return (
+    <View style={{ flex: 1 }}>
+      <ScreenTemplate>
+        <ScrollView ref={scrollViewRef} style={styles.container}>
+          <View style={styles.textContainer}>
+            <Text style={styles.title}>Tell your guests what your place has to offer</Text>
+            <Text style={styles.subtitle}>
+              You can add more amenities after you publish your listing
+            </Text>
+          </View>
+          <View style={styles.rowContainer}>
+            {amenities.map((amenity) => {
+              return (
+                <Chip
+                  key={amenity}
+                  iconName={amenitiesObj[amenity].icon}
+                  iconSet={amenitiesObj[amenity].iconSet}
+                  text={amenitiesObj[amenity].text}
+                  onTouchFunction={() => toggleAmenity(amenity)}
+                  state={selectedAmenities[amenity]}
+                />
+              );
+            })}
+            <View style={{ width: 160 }} />
+          </View>
+          <View style={styles.textContainer}>
+            <Text style={styles.title}>Add other amenities</Text>
+            <Text style={styles.subtitle}>
+              You can put here anything that can make you stand out even more
+            </Text>
+          </View>
+          <View style={styles.rowContainer}>
+            {renderAdditionalChips()}
+            {otherAmenities.length % 2 === 1 && <View style={{ width: 160 }} />}
+          </View>
+          <View style={styles.inputField}>
+            <View style={{ flexDirection: 'row', gap: 10 }}>
+              <Icon name={IconName.Albums} size={24} iconSet="ionicons" />
+              <TextInput
+                value={inputValue}
+                onChangeText={(text) => setInputValue(text)}
+                style={styles.inputText}
+                placeholder="Other amenities"
+              />
+            </View>
+            <TouchableOpacity onPress={() => addOtherAmenities()}>
+              <Icon name={IconName.AddBlank} size={28} iconSet="ionicons" />
+            </TouchableOpacity>
+          </View>
+          <View style={styles.emptySpace} />
+        </ScrollView>
+        <View style={styles.footer}>
+          <Button
+            title="Back"
+            height={35}
+            size={BUTTON_SIZES.SM}
+            type={ButtonType.SECONDARY}
+            onPress={() => {
+              onFormSubmit();
+              console.log('submitted');
+            }}
+          />
+          <Button
+            title="Next"
+            height={35}
+            size={BUTTON_SIZES.SM}
+            type={ButtonType.PRIMARY}
+            onPress={() => {
+              console.log("it's gone");
```

Comment:
All `console.log` used for development purposes should be removed from the PR

Reply:
My bad. Removed